core/themes/basis/template.php

```diff
@@ -44,7 +44,22 @@ function basis_preprocess_layout(&$variables) {
 }
 
 /**
- * Implements template_preprocess_header().
+ * Prepares variables for node templates.
+ *
+ * @see node.tpl.php
+ */
+function basis_preprocess_node(&$variables) {
+  if ($variables['status'] == NODE_NOT_PUBLISHED) {
+    $name = node_type_get_name($variables['type']);
+    $variables['title_suffix'] = array(
```

Comment:
This should make a new item *inside* of `title_suffix`, not replacing the entire thing. With this approach, contextual links would be broken. Instead of `$variables['title_suffix'] = array(` we should use `$variables['title_suffix']['unpublished_indicator'] = array(` or similar.

See contextual_preprocess() for reference.

Reply:
Nice catch. Pushed a new PR with your recommended fix